sailor.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+// phpcs:ignoreFile
+
+declare(strict_types=1);
+
+use Spawnia\Sailor;
```

Comment:
I would appreciate more words about the addition of Sailor.

Although it complements our approach with code generation, it it is not a well known library as I see it.

Reply:
Words? In this file, or md?